docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -81,10 +81,7 @@
     "import numpy as np\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    df0 = df
</pre><p>It looks the original df = table[0] from the previous cell isn't used again and is purely for illustration (case in point, I was having trouble connecting to Wikipedia at the time of this comment but was able to run the notebook by ignoring the previous cell). </p><p> </p><p>If so, I think we can get rid of this line and not have any assignment in the previous cell? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/1132/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
yep, done. found a few other unnecessary assignments/imports as well